pkg/cmd/auth-apiserver/cmd.go

```diff
@@ -39,22 +39,44 @@ func NewOAuthAPIServerOptions(out io.Writer) *OAuthAPIServerOptions {
 	}
 }
 
+func (o OAuthAPIServerOptions) Validate(args []string) error {
```

Comment:
nit: should be a pointer receiver?

Reply:
No, shouldn't. It's must not have side effects.

Reply:
interesting, what can cause side effects?